### PR TITLE
clear up some dev/deploy docs

### PR DESCRIPTION
**Pull request by stonith**
- add prelim section for docker-compose in CONTRIBUTING.md, will need further details for tests etc later
- add link in README.md to the AWS Terraform reference implementation
- add terraform manual commands for AWS

### CODE REVIEW

File: config/infrastructure/aws/README.md [hunk 1]
```diff
@@ -36,9 +36,16 @@ While this infrastructure may be deployed in a number of different ways, this do
...
 - [`ecs-cli`](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI.html) installed and available in your path
 
-## Deploying containers to Fargate with Terraform
+## Deploying to AWS with Terraform
 
-TODO: :smile:
+Expected variables are:
+- `TF_VAR_ecs_task_key_retrieval_env_hmac_key`
+- `TF_VAR_ecs_task_key_submission_env_key_claim_token`
+- `TF_VAR_rds_backend_db_password`
```
Comment (Ginja) [inline config/infrastructure/aws/README.md:44]:
```suggestion
The credentials for AWS Terraform provider are expected to be provided through the standard AWS credential environment variables.

- `AWS_ACCESS_KEY_ID`
- `AWS_SECRET_ACCESS_KEY_ID`

All Terraform variables are defined in `config/terraform/aws/variables.tf` & their values are set in `config/terraform/aws/variables.auto.tfvars`. There are **four** secret variables that should be set through environment variables as to not commit plain text secrets to version control.

- `TF_VAR_ecs_task_key_retrieval_env_ecdsa_key`
- `TF_VAR_ecs_task_key_retrieval_env_hmac_key`
- `TF_VAR_ecs_task_key_submission_env_key_claim_token`
- `TF_VAR_rds_backend_db_password`

If you are using Terraform in Github actions the above can be set as Github secrets, and set as environment variables in your YAML file (see `.github/workflows/terraform.yml`).
```

File: config/infrastructure/aws/README.md [hunk 2]
```diff
@@ -36,9 +36,16 @@ While this infrastructure may be deployed in a number of different ways, this do
...
+
+To run manually:
+1. goto AWS Terraform dir `cd config/terraform/aws`
+1. run `TF_VAR_ecs_task_key_retrieval_env_hmac_key=******* TF_VAR_ecs_task_key_submission_env_key_claim_token="*******=ON" TF_VAR_rds_backend_db_password=******* terraform apply`
```
Comment (Ginja) [inline config/infrastructure/aws/README.md:48]:
```suggestion
1. Go to the AWS Terraform directory - `cd config/terraform/aws`
2. Run 
> TF_VAR_ecs_task_key_retrieval_env_ecdsa_key="******" TF_VAR_ecs_task_key_retrieval_env_hmac_key="******" TF_VAR_ecs_task_key_submission_env_key_claim_token="******" TF_VAR_rds_backend_db_password="******" AWS_ACCESS_KEY_ID="******" AWS_SECRET_ACCESS_KEY="******" terraform [init|plan|apply]
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,6 +28,17 @@ as you'd expect.
 Once you're happy with your changes, please fork the repository and push your
 code to your fork, then open a PR against this repository.
 
+## Development Environment via docker-compose
+
+1. Fork https://github.com/CovidShield/server to your account
+1. Clone your fork of CovidShield/server repo locally by running `git clone git@github.com:<username>/server.git`
```
Comment (jonpulsifer) [inline CONTRIBUTING.md:34]:
```suggestion
1. Clone your fork of CovidShield/server repo locally by running `https://github.com/<username>/server.git`
```

Cloning via HTTP will help those without experience managing SSH keys

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -28,6 +28,17 @@ as you'd expect.
...
+1. Enter the repo directory `cd server`
+1. Run `docker-compose up`
+
+Note: It is normal to see a few errors from the retrieval service exiting initially while the MySQL database is instantiated
```
Comment (jonpulsifer) [inline CONTRIBUTING.md:38]: didn't we add a healthcheck such that this does not happen?

Comment (stonith) [reply CONTRIBUTING.md:38]: Doesn't look like it, we can do that in another PR and then remove this.

**Pull request merged**